src/util/common-utils.ts

```diff
@@ -225,6 +225,10 @@ async function fetchInstalledApps(
   return batchRequests.flat();
 }
 
+
+// To remove the relative path 
+const sanitizePath = (str: string) => str?.replace(/^(\.\.(\/|\\|$))+/, '');
```

Comment:
@cs-raj , Instead of duplicating the same function in each plugin, we can utilise cli-utilities directly once CLI releases with **sanitizePath**. I hope this method will be accessible through cli-utilities?

Reply:
Yes, we can do it, I wrote it here because it was very small

Reply:
okay, no need to create separate ticket for this. We'll handle it in enhancement. Also developerhub url code